Dockerfile

```diff
@@ -9,11 +9,14 @@ COPY requirements.txt /root/swift_request/requirements.txt
 COPY setup.py /root/swift_request/setup.py
 COPY swift_sharing_request /root/swift_request/swift_sharing_request
 
+RUN apk add --no-cache rust cargo \
+    && rm -rf /var/cache/apk/*
+
```

Comment:
This could have been added to the apk command on line 6

Reply:
wanted to keep them separate as these are some special requirements

Reply:
but can as well move